components/engine/integration-cli/docker_cli_swarm_test.go

```diff
@@ -1730,18 +1730,13 @@ func (s *DockerSwarmSuite) TestSwarmServicePsMultipleServiceIDs(c *check.C) {
 
 	// Name Prefix
 	out, err = d.Cmd("service", "ps", "to")
-	c.Assert(err, checker.IsNil)
-	c.Assert(out, checker.Contains, name1+".1")
-	c.Assert(out, checker.Contains, name1+".2")
-	c.Assert(out, checker.Contains, name1+".3")
-	c.Assert(out, checker.Contains, name2+".1")
-	c.Assert(out, checker.Contains, name2+".2")
-	c.Assert(out, checker.Contains, name2+".3")
+	c.Assert(err, checker.NotNil)
+	c.Assert(out, checker.Contains, "no such service: to")
```

Comment:
Alternatively, use the test I wrote at the time of the PR, but which could not be merged upstream due to not being able to update the CLI; https://github.com/moby/moby/pull/32800/commits/4242aeca230f6a0009019be195b5e0195d5ff178

That test also verifies the new behaviour